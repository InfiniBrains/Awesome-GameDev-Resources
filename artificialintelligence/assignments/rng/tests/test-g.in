7 40 9 10
