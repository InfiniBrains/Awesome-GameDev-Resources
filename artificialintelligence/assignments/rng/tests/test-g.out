10
10
10
10
10
9
9
10
10
9
10
9
9
9
10
10
10
9
9
10
9
9
10
9
9
10
9
10
9
10
10
10
9
10
10
9
10
9
10
10
